apps/wallet-mobile/src/features/Swap/common/useSwapTx.ts

```diff
@@ -33,3 +30,34 @@ export const useSwapTx = (options?: UseMutationOptions<YoroiUnsignedTx, Error, {
     ...mutation,
   }
 }
+
+const formatMetadata = (createOrder: Swap.CreateOrderData) => {
+  if (createOrder.selectedPool !== undefined) {
+    return [
+      `address: ${createOrder.address}`,
+      `buyQuantity: ${createOrder.amounts.buy.quantity}`,
+      `buyTokenId: ${createOrder.amounts.buy.tokenId}`,
+      `sellQuantity: ${createOrder.amounts.sell.quantity}`,
+      `sellTokenId: ${createOrder.amounts.sell.tokenId}`,
+      `poolDeposit: ${createOrder.selectedPool.deposit.quantity}`,
+      `poolBacherFee: ${createOrder.selectedPool.batcherFee.quantity}`,
+      `poolId: ${createOrder.selectedPool.poolId}`,
+      `slippage: ${createOrder.slippage}`,
+    ]
+      .map((item) => splitStringInto64CharArray(item))
+      .flat()
+  }
+  return []
```

Comment:
@stackchain  this is still pending as I'm waiting for Krimo to give me the final metadata format that will need to be attached when creating tx